packages/tools/fluid-runner/package.json

```diff
@@ -10,27 +10,57 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
-	"type": "commonjs",
+	"type": "module",
 	"exports": {
 		".": {
-			"types": "./dist/index.d.ts",
-			"default": "./dist/index.js"
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
```

Comment:
I happened to remember Loop importing types from this package, which is why it has alpha-tagged APIs.

Because Loop is using this package, I've temporarily left it dual-format (AB#7518).